neuromation/cli/utils.py

```diff
@@ -466,6 +486,12 @@ def parse_resource_for_sharing(uri: str, root: Root) -> URL:
     return uri_from_cli(uri, root.username, allowed_schemes=("storage", "image", "job"))
 
 
+async def normalize_job_resource(uri: URL, root: Root) -> URL:
+    job_id = await resolve_job(str(uri), client=root.client, default_user=root.username)
+    owner = root.username if not uri.host or uri.host == "~" else uri.host
```

Comment:
AFAIK, we are going to drop tilde-expansion of the username, so let's not extract this logic to a separate method (see same logic in `_parse_as_neuro_image`)

Reply:
`job:/foo` is not the same as `job:foo`. The latter refers to job foo of the current user. The former refers to... all jobs of user foo?